libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,226 @@
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Label>{measuresLocalization.LabelPicker}</Label>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{ dropdown: { width: 150 } }}
+                    id="labelDropdown"
+                    options={labelOptions}
+                    selectedKey={this.state.selectedLabelIndex}
+                    onChange={this.setSelectedLabel}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Feature dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Label>{measuresLocalization.FeaturePicker}</Label>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{ dropdown: { width: 150 } }}
+                    id="featureDropdown"
+                    options={featureOptions}
+                    selectedKey={this.state.selectedFeatureIndex}
+                    onChange={this.setSelectedFeature}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Measure dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Label>{measuresLocalization.MeasurePicker}</Label>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{ dropdown: { width: 200 } }}
+                    id="measureDropdown"
+                    options={measureOptions}
+                    selectedKey={this.state.selectedMeasureIndex}
+                    onChange={this.setSelectedMeasure}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the description of what user chose from dropdowns */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "l1" }}>
+                <Stack.Item>
+                  {/* Empty placeholder so that description lines up with dropdowns */}
+                  <br />
+                </Stack.Item>
+                <Stack.Item>
+                  <FeatureBalanceMeasuresDescription
+                    selectedMeasure={selectedMeasure}
+                    selectedFeature={selectedFeature}
+                    selectedLabel={selectedLabel}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+
+        {/* Renders the description of the chosen measure */}
+        <Stack.Item>
+          <Text style={{ fontWeight: 600 }}>{selectedMeasure} </Text>
```

Comment:
did you try changing this to Label?  that should have same effect as increasing fontWeight

Reply:
Ya, the main issue is that this text is part of a paragraph.
The paragraph format is `<measure name bolded> <measure desc>`. And when we use Label + Text, the two pieces of text don't flow together.

So current implementation looks like this (look at bottom text):
![image](https://user-images.githubusercontent.com/64443771/180870214-2359c02f-bfc5-4a50-b515-210eb0cb2ba4.png)

But when you do Label + Text, you get this:
![image](https://user-images.githubusercontent.com/64443771/180870355-d025736f-cabf-4f99-9dc1-e3befbe55fc1.png)

So then I tried putting the Label + Text within a `<Stack horizontal>`, but it didn't look right either:
![image](https://user-images.githubusercontent.com/64443771/180870582-442a793e-06d3-4702-9dca-4cf6510492f0.png)

Any suggestions on how I can make Label + Text work? I'd prefer this but wasn't sure how

Reply:
hmm, interesting, in my TextHighlighting component I had it set as:

https://github.com/microsoft/responsible-ai-toolbox/blob/main/libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextHighlighting/TextHightlighting.tsx#L28

```<Stack horizontal horizontalAlign="start" tokens={textStackTokens}>```
where:
```
const textStackTokens: IStackTokens = {
  childrenGap: "s2",
  padding: "s2"
};
```
For label I have the style set to:
```
          fontSize: theme.fonts.large.fontSize,
          padding: 0,
```
Not sure why it doesn't work in your scenario, perhaps it's due to the wrapping. Does using horizontalAlign="start" change anything?

Reply:
In trying this suggested, I realized that when a `<Text>` wraps to a newline (due to there being lots of text), the label and text don't render properly.

![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/64443771/180886530-2a50b897-892e-408c-93f3-410531194d24.png)